tests/unit/executors/test_executor.py

```diff
@@ -69,7 +69,37 @@ def test_metas_workspace_replica_peas(tmpdir, replica_id, pea_id):
 def test_executor_workspace_simple(test_metas_workspace_simple):
     executor = Executor(metas=test_metas_workspace_simple)
     assert executor.workspace == os.path.abspath(
-        test_metas_workspace_simple['workspace']
+        os.path.join(
+            test_metas_workspace_simple['workspace'],
+            test_metas_workspace_simple['name'],
+        )
+    )
+
+
+def test_executor_workspace_simple_workspace(tmpdir):
```

Comment:
@JoanFM this ok?

Reply:
what happens if metas workspace and runtime args workspace have different values?

Reply:
Added. Metas has priority